src/app/auth/guards/authorized.guard.ts

```diff
@@ -1,8 +1,27 @@
-import { Injectable } from '@angular/core';
+import { Injectable } from "@angular/core";
+import { Route, Router, UrlSegment, UrlTree } from "@angular/router";
+import { AuthService } from "../services/auth.service";
+import { Observable } from "rxjs";
 
 @Injectable({
-    providedIn: 'root'
+  providedIn: "root",
 })
 export class AuthorizedGuard {
```

Comment:
Well, as per the task requirements this needs to implement the canLoad interface. I'm afraid this is a requirement non-negotiable, although it is deprecated.